speaking-portal-project/src/test/file_check.ts

```diff
@@ -1,6 +1,8 @@
 import { chdir, cwd } from 'node:process'
 import fs from 'fs'
 
+var wavFileInfo = require('wav-file-info')
```

Comment:
ha! This was the module fix that took a bit to realize. I wonder why 'require' statements are still needed for some modules. Will have to look into that for future

Reply:
it might be due to the age of this module, but I would also like to look further into it.